src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -51,6 +51,26 @@ public static ReadOnlyAddressBook getEmptyAddressBook() {
         return tasks.getInternalList();
     }
 
+    /**
```

Comment:
This method isn't needed, you can find task by name using the methods already in the codebase. Furthermore, what if finding by a particular name returns multiple tasks? Your method doesn't take care of that.